contribs/gmf/src/services/querymanager.js

```diff
@@ -3,7 +3,7 @@ goog.provide('gmf.QueryManager');
 goog.require('ol.events');
 goog.require('gmf');
 goog.require('gmf.Themes');
-goog.require('ngeo.Query');
+goog.require('ngeo.query.Query');
```

Comment:
ngeo.query.Service ?  query Query looks weird (In my own opinion)

Reply:
Yes, should be Service.